backend/localplatformlinux.py

```diff
@@ -60,6 +60,8 @@ def chown(path : str,  user : UserType = UserType.HOST_USER, recursive : bool =
     return result == 0
 
 def chmod(path : str, permissions : int, recursive : bool = True) -> bool:
+    if os.geteuid != 0:
```

Comment:
You probably also want to log a warning for future support issues, allowing the project support peeps to understand that it's not running as root, I'd assume.

Reply:
Probably best to just show this once on startup, but yes that is a good idea, will do

Reply:
Sure, I'm about 0% familiar with the codebase, the overall important thing is to ensure you all in support position can identify the situation, now that it can happen.